mindsdb/integrations/handlers/chromadb_handler/chromadb_handler.py

```diff
@@ -314,6 +317,9 @@ def select(
         # always include distance
         if distances is not None:
             payload[TableField.DISTANCE.value] = distances
+
+        # casting embedding to otherwise pandas give error due to multidimensionality
+        payload['embeddings'] = [str(i) for i in payload['embeddings']]
```

Comment:
is it related to only druid handler? or chromadb does't work with LAST with any source?

Reply:
I've just tested with my local pg database
```sql
create knowledge base kb_druid;

CREATE JOB kb_updates_cc AS (
   INSERT INTO kb_druid (
        SELECT id, neighborhood content FROM my_pg.home_rentals where id > LAST
    )
) EVERY 1 min;
```
it works and and when I insert rows in my_pg.home_rentals - they are inserted to KB too by job.

Reply:
yes, you are right @ea-rus